content/applications/hr/fleet/service.rst

```diff
@@ -6,72 +6,284 @@ To properly maintain a fleet of vehicles, regular maintenance as well as periodi
+
+  To add a new tag, type in the tag, then click :guilabel:`Create "tag"`.
+
+  There is no limit to the number of tags that can be selected.
+- :guilabel:`Image`: a photo of either the main contact person or the company logo can be added to
+  the form. Hover over the :guilabel:`image` box in the top-right of the form to reveal a
+  :guilabel:`✏️ (pencil)` icon, and click it. A file explorer window appears. Navigate to the
+  desired file, then click :guilabel:`Open` to select it.
+
+.. image:: service/create-vendor.png
+   :align: center
+   :alt: The top portion of the create vendor form.
+
+Contacts & addresses tab
+************************
+
+After the top-half of the :guilabel:`Create Vendor` form is filled out, add any other contacts and
+addresses associated with the vendor in this tab.
+
+Click the :guilabel:`Add` button to add a new contact, and a :guilabel:`Create Contact` pop-up
+window appears.
+
+Enter the following information on the form:
+
+- :guilabel:`Contact Type`: select the type of contact being added. Click the radio button next to
+  the desired type. The default options are:
+
+  - :guilabel:`Contact`: select this option to add general contact details for employees of the
+    associated vendor.
+  - :guilabel:`Invoice Address`: select this option to add a preferred address for all invoices.
+    When added to the form, this address is selected by default when sending an invoice to the
+    associated vendor.
+  - :guilabel:`Delivery Address`: select this option to add a preferred address for all deliveries.
+    When added to the form, this address is selected by default when delivering an order to the
+    associated vendor.
+  - :guilabel:`Follow-up Address`: select this option to add a preferred address for all follow-up
+    correspondence. When added to the form, this address is selected by default when sending
+    reminders about overdue invoices.
+  - :guilabel:`Other Address`: select this option to add any other necessary addresses for the
+    vendor.
+
+.. image:: service/vendor-contact.png
+   :align: center
+   :alt: The create contact form with all parts filled in.
+
+.. note::
+   If an option *other than* :guilabel:`Contact` is selected for the :guilabel:`Contact Type`, an
+   :guilabel:`Address` section appears on the form. Enter the address details in the
+   :guilabel:`Address` section.
+
+Add any notes to the :guilabel:`Internal notes...` section of the form.
+
+After all of the information is added, click either :guilabel:`Save & Close` to add the one new
+contact, or :guilabel:`Save & New` to add the current address record and create another address
+record.
+
+As contacts are added to this tab, each contact appears in a separate box, with an icon indicating
+what type of contact is listed.
+
+.. example::
+   A :guilabel:`Contact` displays a person icon inside that specific address box, whereas an
+   :guilabel:`Invoice Address` displays a dollar sign icon inside.
```

Comment:
Yes! For some reason, I wanted to have the 'contact' box since that seemed like a default thing people would add- but there are no good emojis for that, nothing even close. So I changed it to a delivery and invoice address, since I could find a truck and dollar bill- and then updated the pic, too.